app.py

```diff
@@ -8,6 +8,10 @@
 import predictions
 import views
 
+import os
+os.environ['PYSPARK_SUBMIT_ARGS'] = '--jars ./libs/spark-als-serialiser_2.11-0.1.jar pyspark-shell'
```

Comment:
i think this should be in the `main` function, just to reduce the possibility of side-effects from importing the file